src/schema/notifications.ts

```diff
@@ -41,6 +41,34 @@ function findDataId(
 	)[0];
 }
 
+function addNewContactGroupToList(array: any, item: any, sortBy: any) {
+	if (sortBy === 'nameAsc' && array.length > 0) {
+		for (let i = 0; i < array.length; i++) {
```

Comment:
We can use lodash/reduce for the following operation.

Reply:
can you please explain on this